### PR TITLE
Reverted incorrect deprecation of JacksonSupport.createInMemoryMapper

### PR DESCRIPTION
**Pull request by shamsasari**

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -206,29 +208,11 @@ Unreleased
 
 * Changes to the JSON/YAML serialisation format from ``JacksonSupport``, which also applies to the node shell:
 
-  * ``Instant`` and ``Date`` objects are serialised as ISO-8601 formatted strings rather than timestamps
```
Comment (shamsasari) [inline docs/source/changelog.rst]: These changes were ported to V3 and so are no longer relevant to V4

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -7,6 +7,8 @@ release, see :doc:`upgrade-notes`.
 Unreleased
 ----------
 
+* ``JacksonSupport.createInMemoryMapper`` was incorrectly marked as deprecated and is no longer so.
```
Comment (mikehearn) [inline docs/source/changelog.rst]: Was it marked as deprecated in the 3.x cycle?

Comment (shamsasari) [reply docs/source/changelog.rst]: yes

Comment (shamsasari) [reply docs/source/changelog.rst]: I think what happened was that the changes were backported into V3 to get the blob inspector out in 3.3/3.2.

**Pull request merged**